fairlearn/reductions/_exponentiated_gradient/_lagrangian.py

```diff
@@ -148,9 +148,16 @@ def _call_oracle(self, lambda_vec):
         signed_weights = self.obj.signed_weights() + self.constraints.signed_weights(lambda_vec)
         if isinstance(self.constraints, ClassificationMoment):
             redY = 1 * (signed_weights > 0)
+            redW = signed_weights.abs()
+        elif isinstance(self.constraints,CDF_DemographicParity):
+            #for the demographic parity in the regression case
```

Comment:
nit: can we leave a space after `#` in comments?

nit: for Demographic Parity as opposed to "for the demographic parity"